src/stores/store/eth1/local/local.go

```diff
@@ -216,6 +217,17 @@ func (s *Store) SignTypedData(ctx context.Context, addr string, typedData *core.
 	return s.Sign(ctx, addr, []byte(encodedData))
 }
 
+func (s *Store) SignEIP191Data(ctx context.Context, addr string, data []byte) (sig, msgHash []byte, err error) {
```

Comment:
`msgHash` is only a keckac(data) of the input data. I would suggest to simplify this interface to only return with `sig` and `error`